tracer/src/Datadog.Trace/Ci/IntelligentTestRunnerClient.cs

```diff
@@ -1174,22 +1165,23 @@ private async Task<string> GetCommitShaAsync()
         return gitSha;
     }
 
-    private async Task<ProcessHelpers.CommandOutput?> RunGitCommandAsync(string arguments, MetricTags.CIVisibilityCommands ciVisibilityCommand, string? input = null)
+    private ProcessHelpers.CommandOutput? RunGitCommand(string arguments, MetricTags.CIVisibilityCommands ciVisibilityCommand, string? input = null)
     {
         TelemetryFactory.Metrics.RecordCountCIVisibilityGitCommand(ciVisibilityCommand);
         try
         {
-            var sw = System.Diagnostics.Stopwatch.StartNew();
-            var gitOutput = await ProcessHelpers.RunCommandAsync(
+            var sw = Stopwatch.StartNew();
+            var gitOutput = ProcessHelpers.RunCommand(
```

Comment:
Shouldn't you be checking for `CommandOutput.TimedOut` in these? e.g. treating it like ExitCode != 0?

Reply:
I'm forcing the exit code to -1 if we timed-out in [8a0c3a7](https://github.com/DataDog/dd-trace-dotnet/pull/6545/commits/8a0c3a71dab76b7f08e6bd24eabba02839f8cc4b) (Because we don't really have an ExitCode)